File: hello/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get input of the height 1 - 8 inclusive
    int h;
    do
    {
    h = get_int("Height: ");
    }
    while(h < 1 || h > 8);
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("input a height:")
    if 0 < height < 9:
        break
    else:
        print("input a number between 1 and 8 inclusive")
        continue

for i in range(height):
    for j in range(height + 2 + i):
        if j == 4 or j == 5 or j < height - (i + 1):
            print(" ", end = "")
        else:
            print("#", end = "")
        print("")
